source/_posts/Integral-Image.md

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
Content should have link to - redirect on Previous blog link also

Reply:
added

Reply:
Done